cmd/publish_remove.go

```diff
@@ -0,0 +1,119 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/aptly-dev/aptly/deb"
+	"github.com/smira/commander"
+	"github.com/smira/flag"
+)
+
+func aptlyPublishRemove(cmd *commander.Command, args []string) error {
+	var err error
+
+	if len(args) < 2 {
```

Comment:
a more user friendly error message would be helpful...

Reply:
on, the other hand, let's not change that now, it should be improoved error messages everywhere..

Reply:
Agreed 👍. I stick to the other command implementation, which are also not very verbose.